modules/phase_field/include/ics/RampIC.h

```diff
@@ -0,0 +1,56 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#ifndef RAMPIC_H
+#define RAMPIC_H
+
+#include "InitialCondition.h"
+#include "InputParameters.h"
+#include "FEProblem.h"
+#include "MooseMesh.h"
```

Comment:
neither MooseMesh.h nor FEProblem.h need to be included in the header. We should avoid unnecessary include statements as they will increase compile time.